R/parameters.R

```diff
@@ -0,0 +1,29 @@
+#' @title Get parameters from SQUIRE model
+#'
+#' @param countryname name of country
+#' @param R0 R0
+#' @param timestep tme step
+#' @param dt dt
+#' @param time_period time period
+#' @param tt_contact_matrix contact matrix time
+#' @param newpopulation population to use instead of actual one
+#' @param numberof_days number of days for the run
+#'
+#' @return SQUIRE parameters
+#' @export
+get_parameters <- function(countryname, R0, timestep, dt, time_period,
```

Comment:
As a general rule, once an argument in a list has a default (here, `newpopulation`) all further arguments should have defaults. Otherwise R's argument matching rules get a bit hard to understand.